react.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+	parser: 'babel-eslint',
+	parserOptions: {
+		ecmaFeatures: {
+			jsx: true,
+		},
+	},
+	extends: [
+		'airbnb',
+		'airbnb/hooks',
+		'./index',
+		'prettier',
+		'prettier/react',
```

Comment:
@nicholasio Curious if you had a look at: https://github.com/yannickcr/eslint-plugin-react before settling for prettier/react

Reply:
This config uses this plugin, but do you see any benefits in using just this plugin directly?

Reply:
Additionally, prettier is way better than eslint to automatically format our code.